bin/crawl-api-links.js

```diff
@@ -0,0 +1,86 @@
+import puppeteer from "puppeteer";
+import core from "@actions/core";
+
+const navigationTimeout = 120000; // Set the navigation timeout to 60 seconds (60000 milliseconds)
+
+function arrayToHTMLList(array) {
+  let html = '<ul>';
+
+  for (let i = 0; i < array.length; i++) {
+    html += '<li>' + array[i] + '</li>';
+  }
+
+  html += '</ul>';
+
+  return html;
+}
+
+async function checkLinks() {
+  const browser = await puppeteer.launch({
+    headless: "new",
+  });
+  const page = await browser.newPage();
+
+  const sitemapUrl = "https://developers.cloudflare.com/sitemap.xml";
+  await page.goto(sitemapUrl, { timeout: navigationTimeout });
+
+  const sitemapLinks = await page.$$eval("url loc", (elements) =>
+    elements.map((el) => el.textContent)
+  );
+
+  const visitedLinks = new Set(); // Set to store visited links
+  const brokenLinks = [];
+
+  for (const link of sitemapLinks) {
+    if (!link) {
+      continue; // Skip if the link is empty
+    }
+
+    await page.goto(link, {
+      waitUntil: "networkidle0",
+      timeout: navigationTimeout,
+    });
+
+    const pageLinks = await page.$$eval("a", (elements) =>
+      elements.map((el) => el.href)
+    );
+
+    for (const pageLink of pageLinks) {
+      if (!pageLink || visitedLinks.has(pageLink)) {
+        continue; // Skip if the pageLink is empty or has already been visited
+      }
+
+      if (pageLink.includes("/api/operations/")) {
```

Comment:
```suggestion
      if (pageLink.includes("developers.cloudflare.com/api/operations/") || pageLink.startsWith("/api/operations/")) {
```
Trying to prevent issues with regular Dev Doc links like `/product/reference/api/operations/create-items/`.

Not sure why all the captured `a.href` URLs are absolute (maybe it's a `puppeteer` thing?), but I'd also test here for relative links.